API.yaml

```diff
@@ -1395,6 +1397,18 @@ paths:
                       type: array
                       items:
                         type: string
+  /ping:
+    get:
+      summary: Get the status of this server
+      tags:
+      - Diagnostic
+      responses:
+        200:
+          description: Should return 1 if the server is alive and ready to respond to requests, 0 otherwise
```

Comment:
how about the server returns 0 for "cannot respond" and a positive integer for can respond. (Number of connections available or something).